docs/packages/connect/methods/cardanoGetPublicKey.md

```diff
@@ -18,6 +18,7 @@ const result = await TrezorConnect.cardanoGetPublicKey(params);
 -   `path` — _required_ `string | Array<number>` minimum length is `3`. [read more](../path.md)
 -   `showOnTrezor` — _optional_ `boolean` determines if publick key will be displayed on device. Default is set to `true`
 -   `derivationType` — _optional_ `CardanoDerivationType` enum. determines used derivation type. Default is set to ICARUS_TREZOR=2
+-   `suppressBackupWarning` - _optional_ `boolean` By default, this method will send a postMessage to show a warning if the wallet does not have a backup. This option suppresses the message.
```

Comment:
this method will send a uiRequest/uiConfirmationRequest?

Reply:
yes, wording could be improved. It would cause connect to emit event (underlying mechanism is not important here)

Reply:
Changed to `By default, this method will emit an event to show a warning if the wallet does not have a backup. This option suppresses the message.`